guides/common/modules/proc_creating-a-custom-ssl-certificate.adoc

```diff
@@ -60,6 +60,22 @@ authorityKeyIdentifier=keyid,issuer
 [ alt_names ]
 DNS.1 = _{context}.example.com_
 ----
+Optional: If you want to add Distinguished Name (DN) details to the CSR, add the following information to the `[ req_distinguished_name ]` section:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+[req_distinguished_name]
+CN = _{context}.example.com_
+countryName =_My_Country_Name_ <1>
+stateOrProvinceName = _My_State_Or_Province_Name_ <2>
+localityName = _My_Locality_Name_ <3>
+organizationName = _My_Organization_Or_Company_Name_
+organizationalUnitName = _My_Organizational_Unit_Name_ <4>
+----
+<1> Two letter code
+<2> Full name
+<3> Full name (example: New York)
+<4> Division responsible for the certificate (example: IT)
```

Comment:
Going by the format we use in this module (line 88), callouts should be ended with a dot. Also, I strongly recommend using **IT department** instead of **IT**, which just feels incomplete.

Ack @apinnick?
```suggestion
<1> Two letter code.
<2> Full name.
<3> Full name (example: New York).
<4> Division responsible for the certificate (example: IT department).
```

Reply:
+1 for "IT department" because otherwise, it looks like a two letter code. :it: